template/express-rest-api/package.json

```
@@ -0,0 +1,32 @@
+{
+  "dependencies": {
+    "express": "^4.17.1",
+    "skuba-dive": "^1.1.0"
+  },
+  "devDependencies": {
+    "@types/express": "^4.17.7",
+    "@types/node": "^14.0.27",
+    "@types/supertest": "^2.0.10",
+    "skuba": "^3.7.7",
+    "supertest": "^4.0.2",
+    "typescript": "^3.9.7"
```

TypeScript is bundled with skuba, and the version gets substituted in on init.

```suggestion
    "skuba": "*",
    "supertest": "^4.0.2"
```

---

Noted, removed it from the dependencies.
How about the skuba should i change that from `"skuba": "^3.7.7"` => `"skuba": "*"`